inspirehep/modules/authors/forms.py

```diff
@@ -274,13 +274,26 @@ class AuthorUpdateForm(INSPIREForm):
 
     """Author update form."""
 
-    # Hidden field to hold author id information
     bai = fields.StringField(
-        widget=HiddenInput()
```

Comment:
Oh, these fields have to be hidden for the users. They should only be visible in the review form.

The following code https://github.com/inspirehep/inspire-next/blob/master/inspirehep/modules/authors/forms.py#L517-L524 was doing the trick, can you check if it works?